internal/pkg/describe/pipeline_status_test.go

```diff
@@ -150,19 +150,18 @@ func TestPipelineStatusDescriber_String(t *testing.T) {
 			testPipelineStatus: &PipelineStatus{*mockPipelineState},
 			expectedHumanString: `Pipeline Status
 
-  Stage             Status              Transition
-  -----             ------              ----------
-  Source              -                   -
-
-  Build             InProgress          ENABLED
-    action1         Failed
-    action2         InProgress
-    action3         Succeeded
-  DeployTo-test     Succeeded           DISABLED
-    action1         Succeeded
-  DeployTo-prod     Failed                -
-    action1         Succeeded
-    TestCommands    Failed
+Stage               Status              Transition
+-----               ------              ----------
+Source                -                   -
+Build               InProgress          ENABLED
```

Comment:
Transitions are from <PreviousStage> to <ThisStage>, right? Like, this transition refers to the Source -> Build transition?

I wonder if there is a way to make this more clear, like placing transition as the first column or adding a `->Transition: ENABLED` line below each stage.

Reply:
Oh--that line is confusing. I was trying to fortify my tests by using the weirdest mock input I could think of. So in that test, there are zero actions within that stage. The "enabled" transition you're seeing is Source -> Build.

But I hear what you're saying, that it could be clearer that it's between stages, not so much part of each. Right now, transition's inclusion within each stage comes from the API call (https://docs.aws.amazon.com/codepipeline/latest/APIReference/API_GetPipelineState.html#API_GetPipelineState_ResponseSyntax).

Reply:
Alternatively transition text could say `(So. -> Bu.) ENABLED` or `(Bu. -> test) ENABLED` or `(test -> prod) DISABLED`

Reply:
Interesting idea! I'll open a GH Issue! Thx.